### ISSUE TITLE
BIAS FX 2 no mouse cursor in REAPER

### ISSUE THREAD

**Issue by a-parhom** [robbert-vdh/yabridge#292]
### Thanks for giving yabridge a shot!

- [X] I read through both the troubleshooting and the known issues sections, and my issue wasn't listed there

### Problem description

BIAS FX 2 plugin has no mouse cursor in REAPER most of the time. 
The cursor can be visible only in one circumstance: when adding the plugin and only if the "Add FX to Track" window is above the place where the actual plugin window will be rendered after clicking the "Add" button. When the plugin window is closed and opened again - the cursor never shows up when the mouse is over it. 
Enabling/disabling DXVK does not help. 
Also, removing "libGLESv2.dll" library from the plugin's Resources folder has no effect at all (though it was needed to show GUI for BIAS FX 1).
Actually, the BIAS FX 1 has kind of "inverted" variant of this issue: it has no cursor when added (almost every time), but reopening it's window resolves that (so not an issue for me).

### What did you expect to happen?

The cursor is visible over the window (at least after reopening it)

### What actually happened?

The cursor never shows over the window, if it was closed and opened again.

### Operating system

Ubuntu 23.10

### How did you install yabridge?

Directly from the release page

### yabridge version

5.1.0

### yabridgectl version

5.1.0

### Wine version

wine-8.20.r0.gf32f8c62 ( TkG Staging Esync Fsync )

### Plugin

BIAS FX 2

### Plugin type

both VST2 and VST3

### Plugin architecture

64-bit

### Host

Reaper 7.07

### Desktop environment or WM

GNOME 45 (Wayland, but X11 behaves the same)

### GPU model

AMD Radeon RX 7600

### GPU drivers and kernel

Mesa 23.2.1 on Linux 6.6.0

**Comment by a-parhom** [comment]
Just noticed: when multiple instances of plugin are open, the cursor does not disappear. So this could be a workaround.

**Comment by robbert-vdh** [comment]
Yeah this is an annoying issue with recent Wine versions and JUCE plugins. It's not an issue with yabridge though, and I can't really do much about it. I cooked up a workaround that sorta worked before but that no longer does. I'll keep this open for a bit so hopefully other people will see it and don't create more issues.

**Comment by magillos** [comment]
It seems to be fixed with Wine 9.1.

**Comment by robbert-vdh** [comment]
I'll check with Wine 9.1 in a bit, thanks! These issues pop up and disappear again with Wine versions. Used to not be an issue, then started being an issue again with Wine 6.x or 7.x, disappeared again for a while, and then it started happening again with some of the later 8.x releases.
